contracts/cw20-atomic-swap/Cargo.toml

```diff
@@ -18,6 +18,7 @@ singlepass = ["cosmwasm-vm/default-singlepass"]
 library = []
 
 [dependencies]
+cw20-escrow = { path = "../cw20-escrow", version = "0.2.0", features = ["library"] }
```

Comment:
I guess it is better to resolve https://github.com/CosmWasm/cosmwasm-plus/issues/73 before finalizing this PR. Packages are getting tangled...

Reply:
It shouldn't be a problem. Just use the `library` feature, as above, when importing stuff from other contracts into your contract.

Reply:
We can always refactor all contracts with the changes proposed in #73 later.

Reply:
Yeah, library does allow that. I agree we should not have that dependency, but rather pull out cw20 coin into cw20 package or such. But happy to merge that and tackle the change in #73